src/types.ts

```diff
@@ -70,3 +71,6 @@ export interface ResponseDkgGetPublicPackage {
 export interface ResponseDkgBackupKeys {
   encryptedKeys: Buffer
 }
+export interface ResponseReviewTransaction {
+  hash?: Buffer
```

Comment:
No need to make this optional, as it always will have a value

Reply:
done!